README.md

```diff
@@ -27,7 +27,7 @@ There are quite a few SBOM formats available today. ```bomber``` supports the fo
 
 ## Providers
 
-```bomber``` supports multiple sources for vulnerability information. We call these *providers*. Currently, ```bomber``` uses [OSV](https://osv.dev) as the default provider, but you can also use the [Sonatype OSS Index](https://ossindex.sonatype.org). 
+```bomber``` supports multiple sources for vulnerability information. We call these *providers*. Currently, ```bomber``` uses [OSV](https://osv.dev) as the default provider, but you can also use the [Sonatype OSS Index](https://ossindex.sonatype.org) or the [Snyk Vulnerability DB](https://security.snyk.io).
```

Comment:
I'd leave README changes out for this moment. DJ is going to split the README, and I'll write updated docs in a follow on PR.